### PR TITLE
Add more information about S3 configuration

### PR DESCRIPTION
**Pull request by ross-spencer**
This commit describes S3 configuration in more detail and provides
information for users about potential pitfalls, e.g. issues that
may be encountered when deleting packages.

Connected to: https://github.com/archivematica/Issues/issues/321
Connected to: https://github.com/archivematica/Issues/issues/696

### CODE REVIEW

File: administrators.rst
Comment (sallain) [inline administrators.rst]:
```suggestion
   If users have stored packages on paths with leading slashes and packages are
```

File: administrators.rst
Comment (sallain) [inline administrators.rst]:
```suggestion
   the slash. Do so with caution and be prepared to put the leading-slash
```

**Review by sallain — COMMENTED** [review]
This looks good, but I confess that I don't 100% understand the path info - I think this is my own limitation rather than the documentation, though! I'd like @dcraze to review it for accuracy before this is merged.

**Comment by ross-spencer** [comment]
Hi @dcraze @sallain there are two PRs against the issue, the other it is hoped will solve the problem outright in the Storage Service: https://github.com/artefactual/archivematica-storage-service/pull/463 

The problem and the workaround are documented here: 

* https://github.com/archivematica/Issues/issues/696#issuecomment-498646029
* https://github.com/archivematica/Issues/issues/696#issuecomment-499026186

I noted in that second link I would make sure there are some docs about this. Perhaps it would be better if I only note the workaround in the `1.9.x` docs? And keep the other information here specifically relevant to `0.14`? (That would be everything except the warning section where I think upgrading to `0.15` will be enough for most users).

**Comment by ross-spencer** [comment]
@sallain I just tried the in-line commits! Thank you for the suggestions. The mechanism works quite well, but I didn't add any useful info to the message. Once this is approved, pending discussion, I'll rebase before committing.

**Comment by sallain** [comment]
@ross-spencer I think you can just note the workaround in the 1.9 docs - no need to go further back.

**Review by sallain — APPROVED** [review]

**Comment by ross-spencer** [comment]
Thanks @sallain for visibility/transparency I've [wordsmithed the commit for `0.15`](https://github.com/artefactual/archivematica-storage-service-docs/pull/23/commits/11b0b18e01a844909beeb2d3d098acd4aef42d15) a little bit to soften the assertion as I think there is a good chance the issue will be gone in `0.15`. But as it's not harmful, might still be useful in the short term for users with S3 who might not go back to look at the correct version for their install. If that's okay, I'll merge this. But I'll get your thumbs-up first. 

And then I've cherry-picked the original [to here](https://github.com/artefactual/archivematica-storage-service-docs/pull/24) for review as well.

**Review by sallain — APPROVED** [review]

**Pull request merged**
